src/hb-ot-layout-common-private.hh

```diff
@@ -479,6 +479,15 @@ struct FeatureParams
     return Null(FeatureParamsSize);
   }
 
+  inline unsigned int get_name_id (hb_tag_t tag) const
+  {
+    if ((tag & 0xFFFF0000u) == HB_TAG ('s','s','\0','\0')) /* ssXX */
+      return u.stylisticSet.uiNameID;
+    if ((tag & 0xFFFF0000u) == HB_TAG ('c','v','\0','\0')) /* cvXX */
+      return u.characterVariants.featUILableNameID;
```

Comment:
One can ask why just returning this and not the other Name IDs. I am asked to do so but I guess that can be done using a separate API and this can support just `featUILableNameID` (or maybe not?)

Reply:
What about something like:
```C
hb_bool_t
hb_ot_layout_get_feature_name_ids (unsigned int* label_id,
                                   unsigned int* tooltip_id,
                                   unsigned int* sample_id,
                                   unsigned int* param_id)
{
  hb_bool_t result = false;

  if (*label_id)
      …;
  if (*tooltip_id)
      …;

  …

  return result;
}
```

Reply:
May be pass also `numNamedParameters` and `character[charCount]` and rename it `hb_ot_layout_get_feature_parameters ()` or something.

Reply:
Done